webapp/web/configuration.css

```diff
@@ -57,12 +57,14 @@
 
 .configure-package-content {
   display: grid;
+  grid-auto-columns: 750px;
+  justify-content: center;
   overflow-y: scroll;
 }
 
 .configure-package-tags {
   display: grid;
-  grid-template-columns: 60% 40%;
+  grid-template-columns: 40% 60%;
```

Comment:
hope this is the right place to comment, but can we have the charts in the package config view a bit more to the right, and so giving a bit more space to the tags? the placeholder image should basically touch the right-most side of the content column

Reply:
Reverted it back to `grid-template-columns: 60% 40%;`. It gives the tags more space